deploy/charts/cert-manager/templates/_helpers.tpl

```diff
@@ -181,8 +181,26 @@ See https://github.com/cert-manager/cert-manager/issues/6329 for a list of linke
 */}}
 {{- define "image" -}}
 {{- $defaultTag := index . 1 -}}
+{{- $global := index . 2 -}}
 {{- with index . 0 -}}
-{{- if .registry -}}{{ printf "%s/%s" .registry .repository }}{{- else -}}{{- .repository -}}{{- end -}}
+{{- if .repository -}}
+    {{- if .registry -}}
+        {{- printf "%s/%s" .registry .repository -}}
+    {{- else -}}
+        {{- .repository -}}
+    {{- end -}}
+{{- else -}}
```

Comment:
There are various ways to consider this...

The changes here cause `<whichever>.image.repository` to be unset, so if someone wants to override, they can set `<whichever>.image.repository` and then optionally set `<whichever>.image.registry`.

I think _most_ users will be quite happy to just set `global.imageRepository` and not need to fiddle w/ anything else.